metricbeat/module/prometheus/collector/_meta/docs.asciidoc

```diff
@@ -134,3 +134,18 @@ To keep only specific metrics, anchor the start and the end of the regexp of eac
   metrics_filters:
     include: ["^node_network_net_dev_group$", "^node_network_up$"]
 -------------------------------------------------------------------------------------
+
+[float]
+=== Setting limits on endpoint sizes
+
+Sometimes endpoints can get too large that they become detrimental to the agent's memory consumption or the underlying storage's stability.
+`max_content_size` can be set to ensure that an endpoint that is larger than a certain size would not get parsed at at. By default there is no set limit.
```

Comment:
```suggestion
`max_content_size` can be set to ensure that an endpoint which's response is larger than a certain size would not get parsed beyond this limit. By default there is no limit set.
```
I think something was not making sense here. Left a quick suggestion but feel free to change it if you prefer something else.